### PR TITLE
refactor(web): transformation of build-bundler into a common es-bundling command 📜

### PR DESCRIPTION
**Pull request by jahorton**
During the big ES-modularization work (🧩) of this (17.0) release cycle, I created a lot of `build-bundler.js` scripts - roughly one per .tsconfig project.  Many of them are _very_ similar to each other.  In #9025, the various configurations were cleaned up and DRY-ed out, but the code _using_ those configurations... that never got reworked and is thus still WET.  Until now.

If this is not tackled at least _somewhat_ now, the problem will continue to proliferate when pursuing separate ES6 artifacts for Web builds - they'd need their _own_ custom versions.  (See #10257.)

As I did with the lm-worker's other two scripts in #10264, the WET `build-bundler` script has been reworked with parameterization.  But, rather than leave it in the `lm-worker`'s space... I have instead implemented this as a common command-script within the `common/web/es-bundling` resource project.  

While none of the other Web projects aside from `lm-worker` utilize it - yet - we should be able to replace most `build-bundler.js` calls, over time, with calls into this common script instead.  Much of this will be done within #10267.

@keymanapp-test-bot skip

### CODE REVIEW

File: common/web/es-bundling/src/common-bundle.mts [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+        break;
+      default:
+        if(!sourceFromArgs) {
+          sourceFromArgs = arg;
+        } else {
+          doHelp(1);
+        }
+    }
+  }
+}
```
Comment (darcywong00) [inline common/web/es-bundling/src/common-bundle.mts:75]: Do you want to `else { doHelp() }` if process.arg.length < = 2 ?

**Review by darcywong00 — APPROVED** [review]
lgtm

File: common/web/es-bundling/tsconfig.json [hunk 2]
```diff
@@ -1,6 +1,9 @@
 {
   "extends": "../../../tsconfig.base.json",
   "compilerOptions": {
+    // Primary settings - the version of ES6 we can target in TS, our downcompile target,
+    // and our module-related settings.
```
Comment (jahorton) [inline common/web/es-bundling/tsconfig.json:5]:
```suggestion
```

Not sure _where_ I meant that comment to be, but I'm sure this place wasn't it.

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 17.0.239-alpha](https://keyman.com/downloads/releases/alpha/17.0.239)

**Pull request merged**